openwisp_monitoring/monitoring/metrics.py

```diff
@@ -0,0 +1,42 @@
+from django.utils.translation import ugettext_lazy as _
+
+from . import settings as app_settings
+
+DEFAULT_METRICS = {
+    'ping': {
+        'label': _('Ping'),
+        'key': 'ping',
+        'field_name': 'reachable',
+        'related_fields': ['loss', 'rtt_min', 'rtt_max', 'rtt_avg'],
```

Comment:
ok, now let's make this information useful.

I would change `Metric.write()` to enforce that `field_name` and `related_fields` are being used as they're configured here.

So that if by mistake a different string is used, the write will fail with an exception (eg: `ValueError`).